main.go

```diff
@@ -99,10 +101,49 @@ func run(p params) error {
 		return errors.Wrap(err, "could not find failed tests")
 	}
 
-	err = j.createIssuesOrComments(failedTests)
+	issues, err := j.createIssuesOrComments(failedTests)
 	if err != nil {
 		return errors.Wrap(err, "could not create issues or comments")
 	}
+	return j.createHtml(issues)
```

Comment:
```suggestion
	return errors.Wrap(j.createHtml(issues), "could not create HTML report")
```

Otherwise there might be too little context.